doc/ref/configuration/master.rst

```diff
@@ -3972,13 +3988,15 @@ A group consists of a group name and a compound target.
       group2: 'G@os:Debian and foo.domain.com'
       group3: 'G@os:Debian and N@group1'
       group4:
-	- 'G@foo:bar'
-	- 'or'
-	- 'G@foo:baz'
+    - 'G@foo:bar'
+    - 'or'
+    - 'G@foo:baz'
```

Comment:
I believe PyCharm did an automatic conversion from tabs to spaces and it jacked this one up... I'll get it fixed.